pkg/slayers/scion_spec.gobra

```diff
@@ -389,7 +395,9 @@ requires acc(&a.(*net.IPAddr).IP, _)
 requires forall i int :: { &a.(*net.IPAddr).IP[i] } 0 <= i && i < len(a.(*net.IPAddr).IP) ==> acc(&a.(*net.IPAddr).IP[i], _)
 requires len(a.(*net.IPAddr).IP) == net.IPv6len
 pure func isConvertibleToIPv4(a net.Addr) bool {
```

Comment:
don't we need termination measures here?

Reply:
good catch. Turns out that there are a few places where we should have added termination measures, but didn't. This seems to be caused by an unsoundness in silicon :o